server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessageListMethodTest.java

```diff
@@ -1898,4 +1900,30 @@ public void getMessageListShouldSortUsingInternalDateWhenNoDateHeader() throws E
     private Date convertToDate(LocalDate localDate) {
         return Date.from(localDate.atStartOfDay(ZONE_ID).toInstant());
     }
+
+    @Test
+    public void getMessageListShouldAcceptLessThan2Pow53NumberForPosition() throws Exception {
+        given()
+            .header("Authorization", aliceAccessToken.serialize())
+            .body("[[\"getMessageList\", {\"position\":" + Number.MAX_LONG_VALUE + "}, \"#0\"]]")
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(200)
+            .body(NAME, equalTo("messageList"));
+    }
+
+    @Test
+    public void getMessageListShouldErrorWhenPositionOver2Pow53() throws Exception {
+        given()
+            .header("Authorization", aliceAccessToken.serialize())
+            .body("[[\"getMessageList\", {\"position\":" + Number.MAX_LONG_VALUE + 1 + "}, \"#0\"]]")
```

Comment:
why MAX_LONG_VALUE? that's MAX_VALUE no? Long.MAX_VALUE is different

Reply:
According to JMAP spec (http://jmap.io/spec-core.html#the-number-datatype) a number can not be greater than 2^53, this is that max value.

Number here being a JMAP object, I feel it's pretty clear that max value is the value from the spec.

Reply:
yes, so that's what I'm saying, it should be MAX_VALUE not MAX_LONG_VALUE.